docs/hw/navqplus_hookup.md

```diff
@@ -0,0 +1,48 @@
+# iRobot® Create® 3 Hookup Guide for NavQ+ by NXP
+
+The iRobot® Create® 3 has a USB-C®[^1] connector implementing a USB 2.0 host capable of 5 V @ 3 A continuous, which can be used to power and communicate with various downstream devices.
+
+NavQ+ is an open source reference design provided by NXP and manufacturable by anyone.
+Since NavQ+ has USB-C® ports capable of being configured as a USB Ethernet Gadget, a cable with a USB-C® connector on both ends is all that is required.
+
+!!! note
+    - Unless set otherwise, the default USB-C® port for use as an Ethernet Gadget device is configured to only the centermost (USB 2) USB-C® as shown in the example pictures.
```

Comment:
... is configured to only the centermost USB-C® port (labeled USB 2) as shown in ...

Reply:
Updated.